src/styles/styles.scss

```diff
@@ -2,6 +2,7 @@
 // Do not remove the comment above - allows other apps to identify the style block
 
 @import "mixins";
```

Comment:
Now that I've looked into this file I wonder why we want to import mixins here. Any thoughts @adamdehaven?

Reply:
We can remove this one; however, instead of importing the mixins into every component, e.g.

```scss
@import '@/styles/mixins';
``` 

We could just add the import to the root vite config here like this:

```ts
additionalData: `
  @import "@kong/design-tokens/tokens/scss/variables";
  @import "@/styles/mixins"; // you need to verify this path properly resolves in the components
`,
```

Reply:
Got it. Done.